app/Http/Controllers/Api/Web/AlbumsController.php

```diff
@@ -83,6 +86,53 @@ public function getShow($id)
         ], 200);
     }
 
+    public function getCachedAlbum($id, $format)
```

Comment:
How would you determine which part of the logic is more appropriate for a model? Should I move everything from `$tracks = $album->tracks;` to `$album->getCachedTrack()`?